[2024-01-30T17:37:12.285+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T16:00:00+00:00 [queued]>
[2024-01-30T17:37:12.321+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T16:00:00+00:00 [queued]>
[2024-01-30T17:37:12.324+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-30T17:37:12.447+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-01-27 16:00:00+00:00
[2024-01-30T17:37:12.458+0000] {standard_task_runner.py:60} INFO - Started process 2598 to run task
[2024-01-30T17:37:12.481+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-01-27T16:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/weather_api_dag.py', '--cfg-path', '/tmp/tmpwx54kwbk']
[2024-01-30T17:37:12.492+0000] {standard_task_runner.py:88} INFO - Job 23: Subtask get_weather_data
[2024-01-30T17:37:12.711+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T16:00:00+00:00 [running]> on host 68c8327b56d3
[2024-01-30T17:37:13.490+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-27T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-27T16:00:00+00:00'
[2024-01-30T17:37:14.763+0000] {logging_mixin.py:188} INFO - {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}
[2024-01-30T17:37:17.449+0000] {logging_mixin.py:188} INFO - {'record': {'cod': 401, 'message': 'Invalid API key. Please see https://openweathermap.org/faq#error401 for more info.'}, 'metadata': {'id': '65b933cddc746540189de3f2', 'createdAt': '2024-01-30T17:37:17.462Z', 'private': True, 'name': 'weather_8ab2169d-9bab-4252-a6c7-89fc33517473', 'collectionId': '65ab4466266cfc3fde7c6f31'}}
[2024-01-30T17:37:17.459+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-30T17:37:17.588+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240127T160000, start_date=20240130T173712, end_date=20240130T173717
[2024-01-30T17:37:17.822+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-30T17:37:17.989+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
